ckan/public/base/css/main.css

```diff
@@ -8095,34 +8095,6 @@ fieldset[disabled] .control-custom.disabled .checkbox.btn.focus {
   background-color: #faedcf;
   margin-right: 3px;
 }
-.resource-upload-field {
-  position: relative;
-  overflow: hidden;
-  display: inline-block;
-  vertical-align: bottom;
-}
-.resource-upload-field label {
-  z-index: 0;
-}
-.resource-upload-field input {
-  filter: alpha(opacity=0);
-  opacity: 0;
-  position: absolute;
-  top: 0;
-  right: 0;
-  z-index: 1;
-  margin: 0;
-  border: solid transparent;
-  border-width: 100px 0 0 200px;
-  cursor: pointer;
-  direction: ltr;
-  -moz-transform: translate(-300px, 0) scale(4);
-}
-.resource-upload-field.loading {
-  display: inline-block;
-  background: url("../../../base/images/loading-spinner.gif") no-repeat center right;
```

Comment:
I guess we can remove this image too!

Reply:
I see it referenced in ckan/public/base/javascript/modules/popover-context.js but it doesn't seem like that's used either (ckan/templates/snippets/context/dataset.html has a class with the same name but that shouldn't use the module IIUC)

Should I remove this .js file too?

Reply:
Yes, I think this can be removed as well